css-layout-classes/index.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="styles.css">
+  <title>css layout classes</title>
+</head>
+
+<body>
+
+  <div class="container">
+
+    <div class="row">
+      <h1>Choose your character!</h1>
```

Comment:
only addition would be a full width column here around the h1

Reply:
gotcha!